ui/package-lock.json

```diff
@@ -28,6 +28,7 @@
         "@types/react-dom": "^17.0.9",
         "@types/react-router-dom": "^5.1.9",
         "@types/yup": "^0.29.13",
+        "@typescript-eslint/eslint-plugin": "^4.31.2",
```

Comment:
I always get this change when running a `./mvnw clean install -DskipTests=true`, so I figured it's finally worth committing it

Reply:
I never got that change, but it look innocent :)

Reply:
I actually see this change with more recent branches too. So +1 to commit it.